components/calendar/src/astronomy.rs

```diff
@@ -1590,8 +1593,8 @@ impl Astronomical {
     /// Based on functions from _Calendrical Calculations_ by Reingold & Dershowitz,
```

Comment:
nit: docs should reflect changes (this fn no longer takes a `Moment`)

Reply:
fixed, thanks